oauth2client/pkce.py

```diff
@@ -0,0 +1,30 @@
+# Copyright 2016 Google Inc. All rights reserved.
```

Comment:
To get all architecturey as I love to do: a code element's specification (doc string, in the case of a function) is for the benefit of its users, and a code element's implementation comments (octothorpe-delimited comments in the body, in the case of a function) is for the benefit of its maintainers. To be without specification, a code element would have to be without users, in which case it would be dead code and would be removed. When a library-private code element has specification, that specification is for its users, and those users are other code elements in the library (and the humans who maintain those other code elements). Just because a code element is not library-public doesn't mean it has zero users and thus can get away without no specification at all.

If someone out there is using "only library-public code elements get documentation" as a practice I think they must really despise their code's future maintainers.